yeonjae/programmers-1/콜라.js

```diff
@@ -0,0 +1,11 @@
+function solution(a, b, n) {
+  let answer = 0;
+  let remainCoke = n;
+
+  while (remainCoke >= a) {
+    answer += Math.floor(remainCoke / a) * b;
+    remainCoke = Math.floor(remainCoke / a) * b + (remainCoke % a);
+  }
+
+  return answer;
+}
```

Comment:
```suggestion
function solution(a, b, n) {
  let answer = 0;
  let remainCoke = n;

  while (remainCoke >= a) {
    const remain = Math.floor(remainCoke / a);
    
    answer += remain * b;
    remainCoke = remain * b + (remainCoke % a);
  }

  return answer;
}
```
- Math.floor(remainCoke / a)가 반복적으로 사용하다보니 변수로 빼는걸 추천드립니다 반복적인 코드를 줄여봐요!!

Reply:
앗 변수로빼주면 더 좋겠네요! 코멘트 감사합니다 :)